br/pkg/lightning/config/config.go

```diff
@@ -552,27 +552,40 @@ type Security struct {
 	KeyPath  string `toml:"key-path" json:"key-path"`
 	// RedactInfoLog indicates that whether enabling redact log
 	RedactInfoLog bool `toml:"redact-info-log" json:"redact-info-log"`
+
+	// this is used to set tls config for lightning in DM
+	// DM may running many lightning instances at same time, so we need to set different tls config name for each lightning
+	TLSConfigName string `toml:"tls-config-name" json:"tls-config-name"`
```

Comment:
seems you didn't change the DSN (`param.Connect()`) to use this `TLSConfigName`

Reply:
is this config something that a user wants to change, or just to be automatically assigned by DM?

if latter i suggest hiding this config from TOML.